lib/util.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Produces an object summarization of limited size. This summarization
+ * does not adhere to the JSON spec so it cannot be reparsed even if
+ * the entire object fits inside the desired size.
+ *
+ * @param {number} n Max length of the summary.
+ * @param {Object} o The object to be summarized.
+ */
+function stringifyPrefix(n, o) {
```

Comment:
Minor nit: it would preferable to have the order of the arguments reversed – that would sound more 'standard' to me, but I won't necessarily be able to dig a up 'standard' documenting that.